client/src/Lobby/SongQueue/SongQueueContainer.js

```diff
@@ -21,7 +21,7 @@ function SongQueueContainer(props) {
 }
 
 const mapStateToProps = (state) => {
-    return state.lobby;
+    return state.lobby, state.player;
```

Comment:
We should probably not combine the two states like that in case state.lobby and state.player have the same state names.

What we should do instead is 
return {
     lobby: state.lobby
     player: state.player
}

This will require you to change whatever's using those props to change like so: props.playing -> props.player.playing

Reply:
Actually, I don't think this component uses the lobby state anymore.  You can probably disregard my previous comment and just delete state.lobby here.